spring-cloud-huawei-service-engine/service-engine-discovery/src/main/java/com/huaweicloud/servicecomb/discovery/registry/ServiceCombRegistration.java

```diff
@@ -35,14 +37,16 @@
  * Spring Cloud Registration的注册过程要求 serviceId 是预先分配好的，service center的注册过程 serviceId 是注册成功
  * 分配的， 两个过程不一样。因此 Registration 只是一个空的实现， 不能够使用， 相关信息在 ServiceRegistry 的实现里面提供。
  */
-public class ServiceCombRegistration implements Registration {
+public class ServiceCombRegistration implements Registration, ServiceRegistration {
```

Comment:
It's not good to change this class. Because you can not change NaocoServiceInstance。 It's better to delegate implementation to this instance.